src/routes/(admin)/questionbank/+page.svelte

```
@@ -160,6 +172,26 @@
+									<div class="flex h-fit flex-col border-t pt-4">
+										{#each question.options as option, optionIndex}
+											{#each Object.entries(option) as [key, value]}
+												<div class="my-auto flex">
+													<span class="bg-primary-foreground m-2 rounded-sm p-3">{key}</span>
+													<p class="my-auto">{value}</p>
+													{#if question.correct_answer.includes(optionIndex)}
+														<CircleCheck class="text-primary my-auto ml-4 w-4" />
+													{/if}
+												</div>
+											{/each}
+										{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)